config/pmd-test.xml

```diff
@@ -110,6 +110,7 @@
     <rule ref="rulesets/java/junit.xml/JUnitTestContainsTooManyAsserts">
         <properties>
             <property name="maximumAsserts" value="11"/>
+            <property name="violationSuppressXPath" value="//MethodDeclaration[@Name='testTokenNumbers'] and [@Name='testRuleNumbers']"/>
```

Comment:
Please put a comment with explanation why this suppression is needed.
You can reuse https://github.com/checkstyle/checkstyle/blob/master/config/pmd-test.xml#L59

Reply:
Done.